### PR TITLE
[SB-678] Tidy up poms

### PR DESCRIPTION
**Pull request by gytis**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,23 +23,26 @@
...
     <arquillian.version>1.4.0.Final</arquillian.version>
-    <junit.version>4.12</junit.version>
-    <spring-boot-bom.version>1.5.13.Final-redhat-3</spring-boot-bom.version>
-    <spring-boot.version>1.5.13.RELEASE</spring-boot.version>
-    <awaitility.version>1.7.0</awaitility.version>
-    <rest-assured.version>2.9.0</rest-assured.version>
+    <awaitility.version>3.1.0</awaitility.version>
     <openjdk18-openshift.version>1.3</openjdk18-openshift.version>
-    <fabric8.generator.from>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift:${openjdk18-openshift.version}</fabric8.generator.from>
+    <openshift-client.version>3.1.8</openshift-client.version>
```
[the remark this answers is not in the document]
Comment (gytis) [reply pom.xml]: Good catch. Not here, this is a copy paste error.

[the remark this answers is not in the document]
Comment (geoand) [reply pom.xml]: Cool

**Comment by gytis** [comment]
@geoand removed the property

**Comment by geoand** [comment]
Thanks!

Maybe we should at some later point, check and see if there is way to enforce that a POM does not contain properties that are not used

**Pull request merged**